src/lightning_lite/accelerators/accelerator.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+import torch
+
+
+class Accelerator(ABC):
+    """The Accelerator base class. An Accelerator is meant to deal with one type of hardware.
+
+    Currently, there are accelerators for:
+
+    - CPU
+    - CUDA (NVIDIA)
+    - MPS (Apple Silicon)
+    - TPU (Google)
```

Comment:
I guess this was copied over, but I suggest we don't list them in a docstring.
This can easily be forgotten, and other accelerators can be registered through the registry

```suggestion
```